bindings/modbus/index.template.html

```diff
@@ -0,0 +1,338 @@
+        that do not support reading or writing on a range of registers (see [[[#example-read-coil-range]]]). 
+        In these circumstances consumers will perform different requests to satisfy the configuration requirements. 
+        <pre id="example-read-coil-range" class="example" title="Read and write 8 single coils with one form">
+                            {
+                                "href": "modbus+tcp://127.0.0.1:60000/1",
+                                "op": [
+                                    "readproperty",
+                                    "writeproperty"
+                                ],
+                                "modbus:entity": "Coil",
+                                "modbus:offset": 1,
+                                "modbus:length": 8
+                            }
+                </pre>
+        Another notable configuration of a form using the modbus vocabulary is the polling mechanism. Thanks
+        to the keyword <code>pollingTime</code> the user can indicate the intervals for observing a particular
+        set of registers. Supposing that the device knows that the value of coil register 1 does change every 
+        1000 ms, in [[[#example-polling]]], it suggest that the polling time should not be faster than 10 ms. WoT 
+        consumers may still create requests faster than the specified time but it should be taken as a reasonable
+        default for observing a property. 
+        <pre  id="example-polling" class="example" title="A complete TD example">
+            {
+                "href": "modbus+tcp://127.0.0.1:60000/1",
+                "op": [
+                    "observeproperty"
+                ],
+                "modbus:entity": "Coil",
+                "modbus:pollingTime: 1000,
+                "modbus:offset": 1
+            }
+        </pre>
+        Finally, [[[#full-td]]] shows a complete device described using modbus ontology.
+        <pre id="full-td" class="example" title="A complete TD example">
+                {
+                  "@context": [
+                      "https://www.w3.org/2019/wot/td/v1",
+                      {
+                          "modv": "https://www.example.com/ns/modbustcp"
+                      }
+                  ],
+                  "title": "ModbusPLC",
+                  "description": "An industrial machine, retrofitted to be IoT capable.",
+                  "id": "uri:dev:ModbusTCPThing",
+                  "securityDefinitions": {
+                      "nosec_sc": {
+                          "scheme": "nosec"
+                      }
+                  },
+                  "security": "nosec_sc",
+              
+                  "base": "",
+                  "properties": {
+                      "limitSwitch1": {
+                          "title": "downLimitSwitch",
+                          "type": "boolean",
```

Comment:
beside the Data schema types we should always include @type and with a precise type annotation like unsignedLong, double, etc. We can rely on XSD

Reply:
from today's TD call: @type is maybe not the best solution. We should evaluate what JSON Schema provides and/or should introduce another term like "subtype".